Packs/PingIdentity/Integrations/PingOne/PingOne.yml

```diff
@@ -0,0 +1,48 @@
+category: Identity and Access Management
+commonfields:
+  id: PingOne
+  version: -1
+configuration:
+- display: Your PingOne Environment ID
+  additionalinfo: "Environment ID is available under the PingOne Dashboard->Environment Properties"
+  name: environment_id
+  required: true
+  type: 4
+- display: PingOne Region
+  additionalinfo: "PingOne has 3 regions, US, EU and Asia"
+  name: region
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: 'Integrates with the PingOne Management API to retrieve, create and delete users'
+display: PingOne
+name: PingOne
+script:
```

Comment:
Please note that you have added only one command configuration. What about the other commands?

Reply:
The new TestPlaybook covers all the commands.

Reply:
My apologies the first commit did not include all changes, not sure what exactly happen. All the commands should be included.